Framework/Algorithms/src/DiscusMultipleScatteringCorrection.cpp

```diff
@@ -197,10 +197,12 @@ void DiscusMultipleScatteringCorrection::exec() {
   }
 
   MatrixWorkspace_sptr sigmaSSWS = getProperty("ScatteringCrossSection");
+  if (sigmaSSWS)
```

Comment:
If this fails does it make sense to continue onwards?
Should we instead have something like:
```
if (!sigmaSSWS){
    throw std::invalid_argument("ScatteringCrossSection was null");
}
```

Alternatively, does validateInputs should pick this up? In which case an assert should be fine here as this shouldn't fail unless a future developer changes validateInputs

Reply:
This is meant to be an optional parameter so I think this is OK. If you look at line 497 there's code to deal with this not being supplied.

```
  if (m_sigmaSS) {
    scatteringXSection = interpolateFlat(m_sigmaSS, kinc);
  } else {
    scatteringXSection = material.totalScatterXSection();
  }
```